### PR TITLE
[DPEDE-913] Add helper message in phone-input

### PR DESCRIPTION
**Pull request by ASanchLumen**
[JIRA Ticket](https://ctl.atlassian.net/browse/DPEDE-913)

### CODE REVIEW

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 1]
```diff
@@ -584,4 +584,34 @@ describe('Phone Input', () => {
...
     });
   });
+
+  describe('Helper message', () => {
+    beforeEach(() => {
+      cy.get('[data-cy="phone-input-danger-helper-message"]')
+        .as("phoneInput");
+      
+    });
+    it("Should initiate with helper-message attribute", function() {
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:594]:
```suggestion
    });
    
    it('Should initiate with helper-message attribute', () => {
```

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 2]
```diff
@@ -584,4 +584,34 @@ describe('Phone Input', () => {
...
+      cy.get("@phoneInput")
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:595]:
```suggestion
      cy.get('@phoneInput')
```

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 3]
```diff
@@ -584,4 +584,34 @@ describe('Phone Input', () => {
...
+        )
+        .find('chi-helper-message').should('exist');
+    })
+
+    it('Should have a danger state helper message', () => {
+      cy.get('@phoneInput')
+        .should('have.attr', 'state', 'danger');
+    
+      cy.get('@phoneInput').find('.chi-label')
+      .should('have.class', '-danger');
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:609]:
```suggestion
      cy.get('@phoneInput')
        .find('.chi-label')
        .should('have.class', '-danger');
```

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 4]
```diff
@@ -584,4 +584,34 @@ describe('Phone Input', () => {
...
+      
+    });
+    it("Should initiate with helper-message attribute", function() {
+      cy.get("@phoneInput")
+        .should(
+          'have.attr',
+          'helper-message',
+          "Optional message"
+        )
+        .find('chi-helper-message').should('exist');
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:601]:
```suggestion
        .find('chi-helper-message')
        .should('exist');
```

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 5]
```diff
@@ -584,4 +584,34 @@ describe('Phone Input', () => {
...
+    })
+
+    it('Should have a danger state helper message', () => {
+      cy.get('@phoneInput')
+        .should('have.attr', 'state', 'danger');
+    
+      cy.get('@phoneInput').find('.chi-label')
+      .should('have.class', '-danger');
+      
+      cy.get('@phoneInput').find('chi-icon')
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:611]:
```suggestion
      cy.get('@phoneInput')
        .find('chi-icon')
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by jllr — APPROVED** [review]

File: cypress/e2e/chi-custom-elements/phone-input.cy.js [hunk 6]
```diff
@@ -584,4 +584,36 @@ describe('Phone Input', () => {
...
     });
   });
+
+  describe('Helper message', () => {
+    beforeEach(() => {
+      cy.get('[data-cy="phone-input-danger-helper-message"]')
+        .as('phoneInput');
+      
+    });
+    it("Should initiate with helper-message attribute", () => {
```
Comment (josecarlosgonzalezv) [inline cypress/e2e/chi-custom-elements/phone-input.cy.js:594]:
```suggestion
    });
    
    it("Should initiate with helper-message attribute", () => {
```

File: src/custom-elements/src/components/phone-input/phone-input.tsx [hunk 7]
```diff
@@ -71,6 +71,10 @@ export class ChiPhoneInput {
    * To define two letter ISO country codes to exclude from Phone input dropdown
    */
   @Prop({ mutable: true, reflect: true }) excludedCountries?: string;
+  /**
+   * To display an additional helper text message below the Textarea
```
Comment (josecarlosgonzalezv) [inline src/custom-elements/src/components/phone-input/phone-input.tsx:75]: Textarea?

```suggestion
   * To display an additional helper text message below the Phone input
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Review by josecarlosgonzalezv — APPROVED** [review]

**Pull request merged**